### PR TITLE
[Makefile] Add jupyter-lab target

### PR DESCRIPTION
**Pull request by ayushkovskiy**
Out of scope of current iteration, but might be still useful.

### CODE REVIEW

File: {{cookiecutter.project_slug}}/Makefile [hunk 1]
```diff
@@ -30,6 +30,9 @@ CUSTOM_ENV_NAME?=image:neuromation-$(PROJECT_POSTFIX)
 
 ##### VARIABLES YOU MAY WANT TO MODIFY #####
 
+# jupyter mode. Available options: notebook, lab.
```
Comment (mariyadavydova) [inline {{cookiecutter.project_slug}}/Makefile]: Jupyter mode. Available options: notebook (to run Jupyter Notebook), lab (to run JupyterLab).

File: {{cookiecutter.project_slug}}/Makefile [hunk 2]
```diff
@@ -288,7 +291,11 @@ jupyter: _check_setup upload-config upload-code upload-notebooks ### Run a job w
 		--volume $(PROJECT_PATH_STORAGE)/$(NOTEBOOKS_DIR):$(PROJECT_PATH_ENV)/$(NOTEBOOKS_DIR):rw \
 		--volume $(PROJECT_PATH_STORAGE)/$(RESULTS_DIR):$(PROJECT_PATH_ENV)/$(RESULTS_DIR):rw \
 		$(CUSTOM_ENV_NAME) \
-		'jupyter notebook --no-browser --ip=0.0.0.0 --allow-root --NotebookApp.token= --notebook-dir=$(PROJECT_PATH_ENV)'
+		'jupyter $(JUPYTER_MODE) --no-browser --ip=0.0.0.0 --allow-root --NotebookApp.token= --notebook-dir=$(PROJECT_PATH_ENV)'
+
+.PHONY: jupyter-lab
```
Comment (mariyadavydova) [inline {{cookiecutter.project_slug}}/Makefile]: jupyterlab

File: {{cookiecutter.project_slug}}/Makefile [hunk 3]
```diff
@@ -288,7 +291,11 @@ jupyter: _check_setup upload-config upload-code upload-notebooks ### Run a job w
...
+jupyter-lab:
```
Comment (mariyadavydova) [inline {{cookiecutter.project_slug}}/Makefile]: Please add help message

**Comment by ayushkovskiy** [comment]
Fixed all comments, thank you @mariyadavydova !

**Pull request merged**